website/content/en/docs/scorecard-alpha/_index.md

```diff
@@ -33,7 +33,7 @@ require if the tests are designed for resource creation.
 
 Scorecard requires the following Kube environment:
 
-- Access to a Kubernetes v1.17.1+ cluster
+- Access to a Kubernetes v1.11.3+ cluster
 
```

Comment:
I think we do not really add here since it is the same pre-requirement defined to use the tool already.

Reply:
agreed, removed it.